### PR TITLE
IBX-4276: Command to revoke user passwords

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-4276
| Versions      | 3.3+

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/users/user_management.md [hunk 1]
```diff
@@ -23,6 +23,36 @@ You can [customize it according to your needs](add_login_form.md#customize-login
 The validity of the password recovery token can be set using the `ibexa.system.<siteaccess>.security.token_interval_spec` parameter.
 By default, it is set to `PT1H` (one hour).
 
+### Revoking passwords
+
+In case of a security situation such as a data leakage, you may need to force users to change their passwords.
+You can do it with the help of the `ibexa:user:expire-password`,
+which revokes the passwords for specific users, User Groups or user users belonging to the chosen Content Type.
```
Comment (webhdx) [inline docs/users/user_management.md:30]: user users 🙂

File: docs/users/user_management.md [hunk 2]
```diff
@@ -23,6 +23,36 @@ You can [customize it according to your needs](add_login_form.md#customize-login
 The validity of the password recovery token can be set using the `ibexa.system.<siteaccess>.security.token_interval_spec` parameter.
 By default, it is set to `PT1H` (one hour).
 
+### Revoking passwords
+
+In case of a security situation such as a data leakage, you may need to force users to change their passwords.
+You can do it with the help of the `ibexa:user:expire-password`,
```
Comment (alongosz) [inline docs/users/user_management.md:29]:
```suggestion
You can do it with the help of the `ibexa:user:expire-password` command,
```
right?

File: docs/users/user_management.md [hunk 3]
```diff
@@ -23,6 +23,36 @@ You can [customize it according to your needs](add_login_form.md#customize-login
...
+- `--iteration-count|-c` - defines how many users are fetched at once. Lowering this value helps with memory issues
+- `--password-ttl|-t` - number of days after which new passwords expire. Used when you update passwords for the whole user Content Type
+
+For example, to revoke the passwords of all users of the `user` Content Type, run:
+
+``` bash
+php bin/console ibexa:user:expire-password --user-content-type-identifier=user --force
+```
+
+To perform a dry run (without saving the results) of rovking passwords of all users from User Group 13, run:
```
Comment (alongosz) [inline docs/users/user_management.md:50]:
```suggestion
To perform a dry run (without saving the results) of revoking passwords of all users from User Group 13, run:
```

File: docs/users/user_management.md [hunk 4]
```diff
@@ -23,6 +23,36 @@ You can [customize it according to your needs](add_login_form.md#customize-login
...
+
+- `--user-id|-u` - the ID of the user. Accepts multiple user IDs
+- `--user-group-id|-ug` - the ID of the User Group. Accepts multiple group IDs
+- `--user-content-type-identifier|-ct` - the identifier of the user Content Type. Accepts multiple Content Types
+
+You can use the following additional options with the command:
+
+- `--force|-f` - commits the change, otherwise the command only performs a dry run
+- `--iteration-count|-c` - defines how many users are fetched at once. Lowering this value helps with memory issues
+- `--password-ttl|-t` - number of days after which new passwords expire. Used when you update passwords for the whole user Content Type
```
Comment (webhdx) [inline docs/users/user_management.md:42]: > Used when you update passwords for the whole user Content Type

This is not correct. This value is used when user content type does not use password expiration and the command has to enable it, then it uses `--password-ttl` value as "Days before password expires" setting in ezuser fieldtype definition.

File: docs/users/user_management.md (hunk 3 above)
Comment (webhdx) [inline docs/users/user_management.md:50]:
```suggestion
To perform a dry run (without saving the results) of revoking passwords of all users from User Group 13, run:
```

**Review by webhdx — CHANGES_REQUESTED** [review]

**Review by webhdx — APPROVED** [review]

**Review by alongosz — APPROVED** [review]

File: docs/users/user_management.md [hunk 5]
```diff
@@ -23,6 +23,36 @@ You can [customize it according to your needs](add_login_form.md#customize-login
 The validity of the password recovery token can be set using the `ibexa.system.<siteaccess>.security.token_interval_spec` parameter.
 By default, it is set to `PT1H` (one hour).
 
+### Revoking passwords
+
+In case of a security situation such as a data leakage, you may need to force users to change their passwords.
+You can do it with the help of the `ibexa:user:expire-password` command,
+which revokes the passwords for specific users, User Groups or users belonging to the chosen Content Type.
```
Comment (dabrt) [inline docs/users/user_management.md:30]:
```suggestion
which revokes the passwords for specific users, User Groups or users that are instances of a specific Content Type.
```

**Review by dabrt — APPROVED** [review]

**Pull request merged**